ci/build_conda.sh

```diff
@@ -15,23 +11,24 @@ export CMAKE_GENERATOR=Ninja
 
 rapids-print-env
 
-rapids-logger "Begin py build"
+# populates `RATTLER_CHANNELS` array and `RATTLER_ARGS` array
+source rapids-rattler-channel-string
 
-CUDA_VERSION="$(cat pynvjitlink/CUDA_VERSION)"
-export CUDA_VERSION
-
-cat > cuda_compiler_version.yaml << EOF
-cuda_compiler_version:
-  - "${CUDA_VERSION}"
-EOF
```

Comment:
This was done to ensure `cuda_compiler_version` matched the intended `CUDA_VERSION` set in that file. That is the one canonical place used to CUDA version in this feedstock. It is also used by wheels to install the right Linux CUDA packages when building as well

Additionally setting `cuda_compiler_version` this way ensured it was considered when generating the package string hash. Maybe `rattler-build` handles this differently

Reply:
I can load it from the file directly and make sure it's part of the package string